R/plot.R

```diff
@@ -77,14 +86,20 @@ enw_plot_obs_by_report <- function(obs, log = TRUE, ...) {
 #' @param posterior A `data.frame` of summarised posterior estimates
 #' containing at least a `confirm` count column a date variable,
 #' quantile estimates for the 5%, 20%, 80%, and 95% quantiles and the
-#' mean and median.
+#' mean and median. This function is wrapped in
+#' [enw_plot_nowcast_quantiles()] and [enw_plot_pp_quantiles()] with sensible 
+#' default labels.
 #'
 #' @return A `ggplot2` plot.
-#'
+#' @seealso [enw_plot_nowcast_quantiles()], [enw_plot_pp_quantiles()]
 #' @family plot
 #' @inheritParams enw_plot_obs
 #' @export
-enw_plot_quantiles <- function(posterior, latest_obs = NULL, log = TRUE, ...) {
+#' @examples
+#' nowcast <- enw_example("nowcast")
+#' nowcast <- summary(nowcast, probs = c(0.05, 0.2, 0.8, 0.95))
+#' enw_plot_quantiles(nowcast, x = reference_date)
+enw_plot_quantiles <- function(posterior, latest_obs = NULL, log = FALSE, ...) {
```

Comment:
feels weird. i get this is MVP, but: quantiles should be discovered, rather than hard coded.

Reply:
That is what I did in `EpiNow2` and it was a very big pain to dev + maintain. I am happy to see this here though I just don't want to do the dev (at least right now).

Reply:
(happy to see an issue on this of course)